src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -73,6 +156,19 @@ def get_by_actor_ids(self, *, actor_ids: List[int]) -> List[APIUser]:
             for u in self.__base_user_query().filter(actor_id__in=actor_ids)
         ]
 
+    def set_option_value(
+        self,
+        *,
+        user: User,
+        key: str,
+        value: Any,
+        project_id: Optional[int] = None,
+        organization_id: Optional[int] = None,
+    ) -> None:
+        UserOption.objects.set_value(  # type: ignore
```

Comment:
This was throwing a type error I didn't understand: `src/sentry/services/hybrid_cloud/user/impl.py:168: error: Access to generic instance variables via class is ambiguous  [misc]`

Reply:
That's because the `objects` Manager class typing isn't, great.